content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -105,6 +105,32 @@ in your **Hosted endpoints** and insert the following data into the pop-up form:
 When you click on **Add endpoint**, your Webhook is configured. You can then click on **reveal** to
 display your signing secret.
 
+Configuration tab
+-----------------
+
+.. _stripe/place_hold_on_card:
+
+Place a hold on a card
+~~~~~~~~~~~~~~~~~~~~~~
+
+Stripe allows you to **capture an amount manually** instead of having an immediate capture. This
+feature earmarks an amount on the customer's account.
+
+.. warning::
+   Only some payments methods support separate authorization and capture.
+   
+   For example, card payments support separating these steps. The expiration window is 7 days.
+
+   With payment methods that don’t support this functionality, like iDEAL, you can’t separate
+   the authorization and capture steps.
+
+.. caution::
+   Odoo doesn't support the partial capture yet. Be aware that if you make a partial capture from
+   Stripe's interface, Odoo will manage it as if it was a full capture.
```

Comment:
```suggestion
   .. caution::
   Odoo doesn't support partial capture yet. Be aware that a partial capture from Stripe's 
   interface will still be managed as a full capture.
```
Here's my suggestion for this part, I just rephrased it, but there was no problem with it to start with x)  
(just try to avoid the future tense, I couldn't manage to take it off x) )

Reply:
I adapted your rephrasing adding 'by Odoo' at the end because you CAN do a partial capture from Stripe, but in Odoo, it will show as if you capture all the amount, even if you only took half the amount.